qutebrowser/commands/runners.py

```diff
@@ -285,6 +286,14 @@ def run(self, text, count=None):
             else:
                 result.cmd.run(self._win_id, args)
 
+            mode_manager = objreg.get('mode-manager', scope='window',
+                                      window=self._win_id)
+            if (result.cmdline[0] != 'repeat-command' and
+                    result.cmdline[0] != 'prompt-accept'):
```

Comment:
Is there any preference between using a list or a tuple to do that? Both are used in other places.